src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/processor/HeaderValidationProcessor.java

```diff
@@ -35,6 +37,12 @@ public void process(Exchange exchange) throws Exception {
 
         RouteProperties routeProperties = (RouteProperties) exchange.getIn().getHeader(MappingConstants.ROUTE_DETAILS);
         String csv = exchange.getIn().getBody(String.class);
+
+        if (ofNullable(csv).isEmpty()) {
+            throw new RouteFailedException(
+                    routeProperties.getFileName() + " file is not found. Please upload file and try again.");
```

Comment:
Use concat rather +?

Reply:
This has been addressed here: https://github.com/hmcts/data-ingestion-lib/pull/10/commits/72e09b4ab03583c76c477a1bff7349d5c5defd1b

_However, it is good to note that `.concat()` will throw an NPE if `routeProperties.getFileName()` returns null_